.github/workflows/add-item-to-project.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Add Issue/PR to Project by team'
+
+on:
+  workflow_call:
+    inputs:
+      project-url:
+        description: 'URL of the GitHub Project where items should be added'
+        required: true
+        type: string
+      team-name:
+        description: 'Team name to match for PR review_requested or requested_team'
+        required: true
+        type: string
+      team-label:
+        description: 'Label that indicates the Issue/PR belongs to the team'
+        required: true
+        type: string
+    secrets:
+      GITHUB_TOKEN:
+        required: true
+
+jobs:
+  add_to_project:
+    name: 'Add to Project Board'
+    runs-on: ubuntu-latest
+
+    # Ensure we have permissions to read issues/PRs and create project issues
+    # (some repos may need 'contents: read', 'issues: write', 'pull-requests: write', etc.)
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: write
+      # If you are using "classic" Projects, also add: projects: write
+
+    steps:
+      - name: Add item to project board
+        uses: actions/add-to-project@244f685bbc3b7adfa8466e08b698b5577571133e
+        if: |
+          (github.event_name == 'pull_request' &&
```

Comment:
It looks like this workflow is set up to respond to an issue or pull request being changed somehow, however, it can only be triggered manually. Is the intent that the author must update `on` with the appropriate events when copying this to a repo?

Reply:
@mcmire This si a shared workflow in the github-tools repo so the trigger will actually be set by the calling github action which will be located in the core repo. Here is a [draft PR](https://github.com/MetaMask/core/pull/5424) that shows that.

Reply:
Oh! I totally forgot you could share workflows like that. Huh. I wonder if we should have been doing this all along rather than publishing actions 🤔 

Anyway, thanks, that makes a lot more sense now.